client/my-sites/site-settings/controller.js

```diff
@@ -78,27 +77,18 @@ module.exports = {
 		}
 
 		// if user went directly to jetpack settings page, redirect
-		if ( site.jetpack && ! config.isEnabled( 'manage/jetpack' ) ) {
+		if ( site && site.jetpack && ! config.isEnabled( 'manage/jetpack' ) ) {
 			window.location.href = '//wordpress.com/manage/' + site.ID;
 			return;
 		}
 
-		if ( ! site.latestSettings || new Date().getTime() - site.latestSettings > ( fiveMinutes ) ) {
-			if ( sites.initialized ) {
-				site.fetchSettings();
-			} else {
-				sites.once( 'change', function() {
-					site = sites.getSelectedSite();
-					site.fetchSettings();
-				} );
-			}
-		}
-
-		const upgradeToBusiness = () => page( '/checkout/' + site.domain + '/business' );
+		const upgradeToBusiness = site
```

Comment:
`upgradeToBusiness` appears to be no longer used here - it's used only in Traffic, so this can be completely removed.